terraform/live/frontend/terragrunt.hcl

```diff
@@ -62,19 +82,19 @@ inputs = {
 
   dynamic_custom_origin_config = [
     {
-      domain_name              = "particule-training.s3-website.eu-west-3.amazonaws.com"
-      origin_id                = "s3-particule-training"
-      origin_path              = ""
-      http_port                = 80
-      https_port               = 443
-      origin_protocol_policy   = "http-only"
-      origin_ssl_protocols     = ["TLSv1", "TLSv1.1" , "TLSv1.2"]
-      custom_headers           = [
-      {
-        name = "Referer"
-        value = local.referer
-      }
-    ]
+      domain_name            = "particule-training.s3-website.eu-west-3.amazonaws.com"
+      origin_id              = "s3-particule-training"
+      origin_path            = ""
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "http-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```

Comment:
is TLSv1.3 available ?

Reply:
Nope and : Choose the security policy that you want CloudFront to use for HTTPS connections. A security policy determines two settings: the SSL/TLS protocol that CloudFront uses to communicate with viewers, and the cipher that CloudFront uses to encrypt the content that it returns to viewers. When you use a custom SSL certificate and SNI, you must use TLSv1 or later. We recommend that you specify TLSv1.2_2019 unless your users are using browsers or devices that do not support TLSv1.2 or later. When you use a custom SSL certificate and dedicated IP addresses, we recommend that you use TLSv1. In this configuration, the TLSv1_2016, TLSv1.1_2016, TLSv1.2_2018 and TLSv1.2_2019 security policies are not available.